### ISSUE TITLE
Add touchpoints surveys to static site and app

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3210]
### Why are we doing this?
The Touchpoints surveys have been created for both the static site and app, and are ready to be added. This ticket tracks the necessary work to add them to both sites.

### Action requested
Touchpoints says the following about adding the custom modal (this is for the app survey): 

> Javascript embed code
> Add the following script to your site to embed your form.
> 
> <script src="https://touchpoints.app.cloud.gov/touchpoints/39dfa04c.js" async></script>
> 

This is the link to the static site survey:
<script src="https://touchpoints.app.cloud.gov/touchpoints/2a6e9509.js" async></script>

There are also direct touchpoints links if we'd rather create our own button, but we should be able to use these script tags to embed a button.

**Comment by Rebecca-LM** [comment]
@emmastephenson can we hold on adding to the public site? We heard from Tim B. we need to get the language approved or modify.

**Comment by emmastephenson** [comment]
@Rebecca-LM sure, we can hold on adding it there. Are we ok to merge in to the app though?

Also, I realized we might get folks replying to this survey when they should really be reaching out to the support team (I can't see this patient in my list, I'm getting an error here, etc.) Do you think it's worth modifying the copy on the form a little to include something about contacting the support team for errors?

**Comment by Rebecca-LM** [comment]
@emmastephenson good call. I modified the language on the doc too, pasting our 1 survey question here. Are you ok with us driving them to the support page vs. contact us page? I figure it's a more self service approach, but could be convinced otherwise.

**What do you think about SimpleReport? Please share feedback on anything you like, or how we can improve.** 

(Looking for help on a current issue? Visit our [support page](https://www.simplereport.gov/support/) instead.)

**Comment by emmastephenson** [comment]
Linking out to the support page seems perfect, thank you!

**Comment by Rebecca-LM** [comment]
Perfect! If you can make that change we're golden.

**Comment by emmastephenson** [comment]
@Rebecca-LM here's the updated survey view. If this looks ok let me know and we'll merge onto the app!

<img width="567" alt="Screen Shot 2022-02-08 at 3 49 39 PM" src="https://user-images.githubusercontent.com/80282552/153095192-aee8a617-db0f-4e74-ac92-1b8cf5f28ca5.png">

**Comment by Rebecca-LM** [comment]
@emmastephenson I was hoping the text in parenthesis could actually go below the first question box and above the "I'm interested in..." checkbox. Lmk if this is possible.

**Comment by emmastephenson** [comment]
@Rebecca-LM yes and no on this one. We can get the text down there as "helper text" but it apparently doesn't support links, so we would lose the href to support:

<img width="581" alt="Screen Shot 2022-02-10 at 3 40 36 PM" src="https://user-images.githubusercontent.com/80282552/153515534-7611bb06-7c19-4348-b759-cf4ee9a6fbf4.png">

**Comment by Rebecca-LM** [comment]
Ok that's fine, we can just have it say to visit our support page w/o the link. Thanks!

**Comment by emmastephenson** [comment]
Ok this is it! Ready to go live? 

<img width="571" alt="Screen Shot 2022-02-10 at 3 48 05 PM" src="https://user-images.githubusercontent.com/80282552/153516203-2386011b-ab73-4616-9930-6dac586b67c4.png">

**Comment by Rebecca-LM** [comment]
@emmastephenson text copy looks good. I'm just looking at font sizes and colors. Is any of that modifiable? 

The text in the last box ("I'm interested in..." looks bigger than the first, main question. The question should be the biggest size and the "I'm interested in" should be smaller). Are we also able to make the parenthetical text about support a lighter color, like a dark grey?

**Comment by emmastephenson** [comment]
@Rebecca-LM no, it looks like we don't have any permissions to update the font or styling. For that we'd need to start from scratch and create an entirely new custom form, rather than this default version

**Comment by Rebecca-LM** [comment]
Ok then let's keep as is and push live! Ty!

**Comment by emmastephenson** [comment]
Merged! Should be live within the hour 👀

**Comment by Rebecca-LM** [comment]
Update on static site survey: Tim b. suggested sending the draft to Jeffery M Zirger (cc: Tim) at the CDC for approval on the custom copy. We’re going to wait until the sentiment analysis stuff is ironed out with Tim via a meeting scheduled for March 1. Then we'll send it all in for approval.

cc: @lg-king

**Comment by emmastephenson** [comment]
Waiting to re-release for PRA review.

Once we do enable, we need to make sure this doesn't block the submit button on mobile for patient results.

**Comment by Rebecca-LM** [comment]
@vz3 lmk when you've had a pass of the PRA form we need to submit for this

**Comment by Rebecca-LM** [comment]
PRA form request and draft questions sent to CDC March 10

**Comment by emmastephenson** [comment]
@Rebecca-LM awesome! We should be ok to re-enable on the static site immediately, but we'll need to do some additional customization on the app side to make sure Touchpoints doesn't block the patient experience (previously it was blocking the view results button). 
Did they approve the existing form questions, or do we have to make any edits?
This is the static site form:
![Screen Shot 2022-04-05 at 11 28 19 AM](https://user-images.githubusercontent.com/80282552/161824910-dead7420-e128-43a5-ab9c-2db63e416d76.png)
![Screen Shot 2022-04-05 at 11 28 31 AM](https://user-images.githubusercontent.com/80282552/161824929-a555a639-05a6-4ef1-9fd0-1e24c476e51c.png)

And this is the app form:
![Screen Shot 2022-04-05 at 11 28 50 AM](https://user-images.githubusercontent.com/80282552/161824964-b23f6a32-358a-4c13-8b6d-96ff96e19056.png)

**Comment by Rebecca-LM** [comment]
Here is the updated source of truth doc: https://docs.google.com/document/d/1c_svdghAC2mvCH09PVvOusLhWSfPuudmfV3M8mLKOu0/edit.

Basically we only added one question:

Select how strongly you agree or disagree with the following statement:
I think SimpleReport is easy to use.

- Strongly agree
- Agree
- Neither agree nor disagree
- Disagree
- Strongly disagree

**Comment by emmastephenson** [comment]
Closing in favor of #3625 and #3626
